lib/ansible/modules/cloud/amazon/execute_lambda.py

```diff
@@ -262,7 +262,7 @@ def main():
 
     if invoke_params['InvocationType'] == 'RequestResponse':
         try:
-            results['output'] = json.loads(response['Payload'].read())
+            results['output'] = json.loads(response['Payload'].read().decode('utf8'))
```

Comment:
Guessing it should also probably be using `ansible.module_utils._text` too, instead of a direct decode.